queue_services/entity-emailer/src/entity_emailer/email_processors/__init__.py

```diff
@@ -61,23 +61,31 @@ def get_recipients(option: str, filing_json: dict, token: str = None) -> str:
                         break
             recipients = f'{recipients}, {comp_party_email}'
     else:
-        headers = {
-            'Accept': 'application/json',
-            'Authorization': f'Bearer {token}'
-        }
         identifier = filing_json['filing']['business']['identifier']
-        if not identifier[:2] == 'CP':
-            # only add recipients if not coop
-            contact_info = requests.get(
-                f'{current_app.config.get("AUTH_URL")}/entities/{identifier}',
-                headers=headers
-            )
-            contacts = contact_info.json()['contacts']
-            if not contacts:
-                logger.error('Queue Error: No email in business profile to send output to.', exc_info=True)
-                raise Exception
-
-            recipients = contacts[0]['email']
+        recipients = get_recipient_from_auth(identifier, token)
+
+    return recipients
+
+
+def get_recipient_from_auth(identifier: str, token: str) -> str:
+    """Get the recipients for the email output from auth."""
+    recipients = ''
+    headers = {
+        'Accept': 'application/json',
+        'Authorization': f'Bearer {token}'
+    }
+    if not identifier[:2] == 'CP':
```

Comment:
So this will not send AR reminders to our existing Coops?

Reply:
This was an existing implementation. I will refactor to exclude this condition for AR reminders.